pages/bestellung/index.js

```diff
@@ -1,91 +1,42 @@
 import styled from "styled-components";
 import Link from "next/link";
+import Breadcrumb from "../../components/Breadcrumb";
+import ZurueckButton from "../../components/Zurueckbutton";
+import StyledBody from "../../components/StyledBody";
+import DefaultButton from "../../components/DefaultButton";
 
 export default function BestellungPage() {
+  const path = "/online bestellen und kaufen";
   return (
     <div>
```

Comment:
You can use a fragment here 
```suggestion
    <>
```

Reply:
or something else with more semantic value